js/form.js

```diff
@@ -1,5 +1,80 @@
+import '../pristine/pristine.min.js';
 import { disableForm, enableForm } from './utils.js';
 const form = document.querySelector('.ad-form');
 
-disableForm(form);
-enableForm(form);
+const disableAdForm = () => {
+  disableForm(form, 'ad-form--disabled');
+};
+
+const enableAdForm = () => {
+  enableForm(form, 'ad-form--disabled');
+};
+
+export { disableAdForm, enableAdForm };
```

Comment:
Если множественный экспорт, то обычно он должен быть в самом низу файла и "собирать" все что нужно экспортировать, иначе можно экспортировать каждую функцию отдельно
```suggestion
export const disableAdForm = () => {
  disableForm(form, 'ad-form--disabled');
};

export const enableAdForm = () => {
  enableForm(form, 'ad-form--disabled');
};
```

Потому что если ты сейчас укажешь его тут а ниже добавишь новую функцию на экспорт, тебе придется переносить этот экспорт вниз в любом случае